core/server/models/user.js

```diff
@@ -110,6 +120,29 @@ User = ghostBookshelf.Model.extend({
             })();
         }
 
+        /**
+         * CASE: add model, hash password
+         * CASE: update model, hash password
+         *
+         * Important:
+         *   - Password hashing happens when we import a database
```

Comment:
We rehash the password and set the account to locked, so that users have to reset their password to login. It would be super weird to be able to import a password directly?